scripts/linters/general_purpose_linter.py

```diff
@@ -367,13 +367,6 @@
         'excluded_files': (),
         'excluded_dirs': ()
     },
-    {
-        'regexp': re.compile(r'datetime.datetime.now\(\)'),
-        'message': 'Please use datetime.datetime.utcnow() instead of '
-                   'datetime.datetime.now().',
-        'excluded_files': (),
-        'excluded_dirs': ()
-    },
     {
         'regexp': re.compile(r'ndb\.'),
```

Comment:
Have a separate DisallowedModulesCheckers check. (I'll create a create issue for this)